Description: Don't show the gnome-settings-daemon autostart in the session capplet, the description is confusing and it's a required component
Bug: https://launchpad.net/bugs/146918

diff -Nur gnome-settings-daemon-2.29.92/data/gnome-settings-daemon.desktop.in.in gnome-settings-daemon-2.29.92.ubuntu/data/gnome-settings-daemon.desktop.in.in
--- gnome-settings-daemon-2.29.92/data/gnome-settings-daemon.desktop.in.in	2009-05-03 20:15:30.000000000 +0200
+++ gnome-settings-daemon-2.29.92.ubuntu/data/gnome-settings-daemon.desktop.in.in	2010-03-10 00:28:52.000000000 +0100
@@ -3,6 +3,7 @@
 _Name=GNOME Settings Daemon
 Exec=@LIBEXECDIR@/gnome-settings-daemon
 OnlyShowIn=GNOME;
+NoDisplay=true
 X-GNOME-Autostart-Phase=Initialization
 X-GNOME-Autostart-Notify=true
 X-GNOME-AutoRestart=true
